azurelinuxagent/common/protocol/hostplugin.py

```diff
@@ -136,11 +138,15 @@ def get_artifact_request(self, artifact_url, artifact_manifest_url=None):
         headers = {HEADER_VERSION: API_VERSION,
                    HEADER_CONTAINER_ID: self.container_id,
                    HEADER_HOST_CONFIG_NAME: self.role_config_name,
-                   HEADER_ARTIFACT_LOCATION: artifact_url}
+                   HEADER_ARTIFACT_LOCATION: artifact_url,
+                   HEADER_VERIFY_FROM_ARTIFACTS_BLOB: "true"}
```

Comment:
I see that this header is added unconditionally. Will Host GA Plugins that do not support Fast Track handle it correctly? (i.e. ignore it rather than choke on it)

Reply:
Legacy HostGAPlugins will ignore it

Reply:
thanks